README.md

```diff
@@ -558,6 +597,35 @@ Add the `$ROBOTOLOGY_SUPERBUILD_ROOT/build/install/lib/python2.7/dist-packages`
 ### Check the installation
 The folder mentioned in the configuration section should contain `*.py` files which correspond to the generated python bindings. Open a python interpreter and try to import modules.
 
+## Oculus
+Support for this dependency is enabled by the `ROBOTOLOGY_USES_OCULUS_SDK` CMake option.
+
+**Warning: at the moment the Oculus SDK does not support macOS and Linux, so this option is only supported
+on Windows.**
+
+### System Dependencies
+To check and install the dependencies please follow the steps for Oculus SDK mention [here](https://github.com/robotology/walking-teleoperation/blob/master/docs/Dependencies.md). 
+
+### Configuration
+To configure please follow the steps for Oculus SDK mention [here](https://github.com/robotology/walking-teleoperation/blob/master/docs/Dependencies.md).
+
+### Check the installation
```

Comment:
If there is not "check the installation" step, I would remove the section (similarly for Cyberith).

Reply:
Done!